tests/integration-tests/tests/cloudwatch_logging/test_cloudwatch_logging.py

```diff
@@ -584,51 +575,41 @@ def get_config_param_vals(cw_logging_enabled):
         )
     )
     queue_size = int(environ.get("CW_LOGGING_QUEUE_SIZE", 1))
-    return {"enable": str(cw_logging_enabled).lower(), "retention_days": retention_days, "queue_size": queue_size}
-
-
-@pytest.mark.parametrize("cw_logging_enabled", [True, False])
-@pytest.mark.parametrize("cw_logs_persist_after_delete", [True, False])
-@pytest.mark.regions(["us-east-1", "cn-north-1", "us-gov-west-1"])
-@pytest.mark.instances(["t2.micro", "c5.xlarge"])
-def test_cloudwatch_logging(
-    region,
-    scheduler,
-    instance,
-    os,
-    pcluster_config_reader,
-    clusters_factory,
-    cw_logging_enabled,
-    cw_logs_persist_after_delete,
-):
+    return {"enable": "true", "retention_days": retention_days, "queue_size": queue_size}
+
+
+@pytest.mark.regions(["us-east-1"])
+@pytest.mark.oss(["centos7", "ubuntu1804", "alinux2"])
+@pytest.mark.instances(["c5.xlarge"])
+def test_cloudwatch_logging(region, scheduler, instance, os, pcluster_config_reader, clusters_factory):
     """
     Test all CloudWatch logging features.
 
     All tests are grouped in a single function so that the cluster can be reused for all of them.
     """
     environ["AWS_DEFAULT_REGION"] = region  # So that it doesn't have to be passed to boto3 calls
-    config_params = get_config_param_vals(cw_logging_enabled)
+    config_params = get_config_param_vals()
     cluster_config = pcluster_config_reader(**config_params)
     cluster = clusters_factory(cluster_config)
     test_runner = CloudWatchLoggingTestRunner(
-        _get_log_group_name_for_cluster(cluster.name),
-        cw_logging_enabled,
-        config_params.get("retention_days"),
-        cw_logs_persist_after_delete,
+        log_group_name=_get_log_group_name_for_cluster(cluster.name),
+        enabled=True,
+        retention_days=config_params.get("retention_days"),
+        logs_persist_after_delete=True,
```

Comment:
wondering if we should have this set to False so that we test the main use case and the proper logs cleanup. At most we could have a separate test that does not iterate on all dimensions (but eventually includes awsbatch) only to test log retention

Reply:
Why not test this as well? I don't think it adds a lot of time given that the delete operation will need to take place anyways.